BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+
+**M. Àngels** -:  So I see you maintain a conversation. So from my understanding, Nicolas will need to make a separate, right?
+
+**Matthias** -: Yes, indeed.
+
+**M. Àngels** -:  So shall we add that we are waiting for the author?
+
+**Matthias** -: I think that's pretty much the case now because the proposal by itself...
+
+**Robert** -: [Issues with the audio] I don't think based on what Matthias was saying before, which he'd probably be staying now, if he could. It sounds like that can only really go ahead as a separate CIP because of the existing implementations that it would be incompatible with.
+
+**M. Àngels** -:  And apologies on my lack of knowledge on this. So in this case, what 
+is the next step.
+
+**Robert** -: As far as I know, we would say that this is waiting for author. And then 
+if that is the agreement, then we would deprecate this one in favor of the new PR once the author submits it.
+
+**M. Àngels** -:  Okay.
+
+**Matthias** -: I think it can also be the same PR right, it can just modify it.
+
+**Robert** -: I can add that label now waiting for author, if that's okay with everybody else.
+
+**Matthias** -: Yeah, good.
+
+**M. Àngels** -:  I was doing it already, so don't worry about this.
+
+**Matthias** -: So the content itself, I mean, I already admitted my remark last week and I agree with the author. I think I've just asked him to put that as a rational because I would avoid having this discussion again. Should be fine, but just as a separate content.
+
+**M. Àngels** -:  Perfect.
+
+**Matthias** -: Two different.
+
+### Last Check 
+
+#### PR263
+[PR263 CIP-0054? | Cardano Smart NFTs](https://github.com/cardano-foundation/CIPs/pull/263)
+
+Okay. So we move to the items on last check. So we have PR 263, CIP 54, Cardano smart NFTs.
+
+**Matthias** -: Yes. So this one was on last check, I think because it was substantially discussed on the forum. And this is like the results of quite a few weeks of discussions. Had another go at it today, actually. So made a few small remarks. It's mainly on the editorial aspect. So just titles and sections to stick a bit more to [crosstalk 00:03:17].
+
+**Robert** -: [Issues with the audio] I was hoping again, that Matthias was going to be jumping in with his eloquent voice because he's already said everything. He's made some substantial comments on this issue so far. And I think what he wrote today on GitHub is self explanatory for the things that the author would have to respond to still. I guess this could remain in last check until we have responses about all of those. So I would suggest postponing this until the next meeting.
```

Comment:
```suggestion
**Robert** -: [Issues with the audio; Robert still cannot hear Matthias] I was hoping again, that Matthias was going to be jumping in with his eloquent voice because he's already said everything. He's made some substantial comments on this issue so far. And I think what he wrote today on GitHub is self explanatory for the things that the author would have to respond to still. I guess this could remain in last check until we have responses about all of those. So I would suggest postponing this until the next meeting.
```

Reply:
> I was hoping again, that Matthias was going to be jumping in with his eloquent voice

:smile: !